waypointList waypointList
---
bool result
---
#progress is in percent
uint8 latitude_progress
uint8 longitude_progress
uint8 altitude_progress
uint8 index_progress
